### PR TITLE
Office-js: Correcting documentation for TableBinding.setFormatsAsync

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://dev.office.com/reference/add-ins/shared/binding.tablebinding.setformatsasync
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/office-js/index.d.ts [hunk 1]
```diff
@@ -3122,18 +3122,117 @@ declare namespace Office {
...
+         * **Supported ranges in cells property**
+         * 
+         * <table>
+         *   <tr>
+         *     <th>cells range settings</th>
+         *     <th>Description</th>
+         *   </tr>
+         *   <tr>
+         *     <td>{row: n}</td>
+         *     <td>Specifies the range that is the nth row of data in the table.</td>
```
Comment (Rick-Kirkham) [inline types/office-js/index.d.ts]: Should be "Specifies the range that is the zero-based nth row of data in the table."

File: types/office-js/index.d.ts [hunk 2]
```diff
@@ -3122,18 +3122,117 @@ declare namespace Office {
...
+         *   </tr>
+         *   <tr>
+         *     <td>{column: n}</td>
+         *     <td>Specifies the range that is the nth column of data in the table.</td>
```
Comment (Rick-Kirkham) [inline types/office-js/index.d.ts]: Should be "Specifies the range that is the zero-based nth column of data in the table."

File: types/office-js/index.d.ts [hunk 3]
```diff
@@ -3122,18 +3122,117 @@ declare namespace Office {
...
+         *   </tr>
+         *   <tr>
+         *     <td>{row: i, column: j}</td>
+         *     <td>Specifies the range of cells from the ith row to the jth column of data in the table.</td>
```
Comment (Rick-Kirkham) [inline types/office-js/index.d.ts]: Should be "Specifies the single cell that is the ith row and jth column of the table."

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

File: types/office-js/index.d.ts (hunk 3 above)
Comment (AlexJerabek) [reply types/office-js/index.d.ts]: @Rick-Kirkham All fixed.

**Review by Rick-Kirkham — APPROVED** [review]

**Pull request merged**